internal/command/kubernetes.go

```diff
@@ -0,0 +1,384 @@
+package command
+
+import (
+	"context"
+	"crypto/rand"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+	"time"
+
+	"github.com/fyne-io/fyne-cross/internal/cloud"
+	"github.com/fyne-io/fyne-cross/internal/log"
+	"github.com/fyne-io/fyne-cross/internal/volume"
+
+	v1 "k8s.io/api/core/v1"
```

Comment:
This is not a descriptive name. Why not "k8s" instead? or "core" which is what it would use with no name attached I think.

Reply:
I followed here what other project using the Kubernetes API in go where doing, but I agree going core and meta make sense.